php-checks/src/test/java/org/sonar/php/checks/phpunit/NoTestInTestClassCheckTest.java

```diff
@@ -28,4 +28,9 @@ class NoTestInTestClassCheckTest {
   void test() {
```

Comment:
Maybe you can also rename this method to show what is the difference between both test cases.